Created new effect, "Anti-Homing shots"

BMStrom opened the pull request:
Created new effect, "Anti-Homing shots" that applies the Anti-Homing effect to the player's projectiles

BMStrom committed:
Created new effect, "Anti-Homing shots"

Created new effect, "Anti-Homing shots" that applies the Anti-Homing effect to the player's projectiles

BMStrom committed:
Added "Random Projectile Effect"

Added "Random Projectile Effect," which applies a random effect (currently chosen between "Linear Arc Shots" and "Anti-Homing Shots") to the player's projectiles

BMStrom committed:
Updated manifest

Updated manifest.json to account for new features ("Anti-Homing Shots" and "Random Projectile Effect")

Miczu commented:
Can you make the possible effects into an array, so we are not locked just to effects with your nickname in it?
It would bloat the votting pool even more if we have both anti homing and random projectile. Please remove anti homing as a separate vote option and keep its effect in random projectile effect.

Have you considered adding some other effects into random effect pool from existing votes to make it bigger than 2 options at release?

Also it feels bad if chat votes for something and they roll "nothing", so that scenario should be removed.

BMStrom commented:
Which effects do you think I should put in there? Should I still have Linear Arc Shots as a separate option?
The ideas I have for which ones to put in the effect pool are these:

anti-homing shots
ceasefire - not quite sure about this one, since it's a wand effect rather than a projectile effect
everyone loves larpa - not sure about this one because it affects enemy projectiles as well
linear arc shots
moneyshot
spells to worms - not sure about this one because it affects enemy projectiles as well
teleport nullification? - this one affects all teleportation methods, not just projectiles

Miczu commented:
I think that would be a solid list, this would keep chat entertained and you/anyone can add more to it without worry about overflowing votes with one category of effects.

BMStrom committed:
Updated Random Projectile Effect

Removed "Anti-Homing Shots" effect as a separate effect and added it to the "Random Projectile Effect" twitch outcome file, added more options to "Random Projectile Effect"

BMStrom committed:
Updated probability of Linear Arc

Updated Linear Arc Shots probability to what it was before

BMStrom committed:
Updated manifest

Updated manifest.json to reflect changes to "Random Projectile Effect" and "Anti-Homing shots"

Miczu committed:
Update manifest.json

Miczu reviewed (approved):
Works very well!
